portfolio/src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,7 @@
+package com.google.sps.data;
+
+public class Constants {
+    private Constants() {}
```

Comment:
Do we need this constructor?

Reply:
You're right, Java provides a default constructor and I never create a Constants object anyway. I'll remove it